client/src/pages/Profile.js

```diff
@@ -48,59 +48,59 @@ const useStyles = makeStyles((theme) => ({
     }
 }))
 
-const tempData = [
-    {
-        image: 'tattoo-1.png',
-        noSketches: '24',
-        title: 'Lion tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '150'
-    },
-    {
-        image: 'tattoo-3.png',
-        noSketches: '24',
-        title: 'Cat tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '200'
-    },
-    {
-        image: 'tattoo-7.png',
-        noSketches: '24',
-        title: 'Lion tattoo concept in minimal style',
-        description: 'Looking for cool simplicity ideas of lion',
-        prizeAmount: '90'
-    },
-]
+const user_id = 1
+const hostname = "http://localhost:5000"
 
 export default function Profile() {
     const classes = useStyles()
     const [activeTab, setActiveTab] = useState(0)
+    const [inProgressContestCards, getInProgressContestCards] = useState([])
+    const [completedContestCards, getCompletedContestCards] = useState([])
 
     const handleTabChange = (event, newActiveTab) => {
         setActiveTab(newActiveTab)
     }
+    useEffect(() => {
+        const xhr = new XMLHttpRequest()
```

Comment:
you should use fetch and put your fetch function inside apiCall.js to keep consistency